packages/trader/src/App/Components/Elements/PositionsDrawer/positions-modal-card.jsx

```diff
@@ -72,7 +72,9 @@ const PositionsModalCard = ({
                         icon={contract_info.underlying ? `IcUnderlying${contract_info.underlying}` : 'IcUnknown'}
                         size={34}
                     />
-                    <span className='positions-modal-card__symbol'>{contract_info.display_name}</span>
+                    <Text size='xxxs' className='positions-modal-card__symbol'>
```

Comment:
3xs?

Reply:
This is inside the theme.scss. :D beleive or not, we have it. :D